app/main.py

```diff
@@ -1,3 +1,19 @@
-def shop_trip():
-    # write your code here
-    pass
+from app.customers import Customer
+
+
+def shop_trip() -> None:
+
+    data = Customer.get_data()
+    customers = []
+
+    for customer_data in data["customers"]:
+        customer = Customer(customer_data["name"],
```

Comment:
After opening the parentheses, write each the arguments on a new line, including the first one, the closing bracket is also written on a new line